docs/UserGuide.md

```diff
@@ -176,7 +176,7 @@ As members leave the club, you can delete a member along with all of his/her con
 Format: `deleteMember NAME`
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-The NAME you specify must be in the currently displayed list of Members  
+The NAME specified is case-sensitive and must be in the current displayed list of Members.
```

Comment:
I think we can make this clearer by making it "The NAME specified must be in the currently displayed list of Members when executing this command". We also specified above the NAME field is case-sensitive so I guess we don't have to repeat it?